docs/contributing.md

```diff
@@ -13,11 +13,19 @@ If any notional data is required for the test ensure an appropriately titled fol
 If you would like to run unit tests for the project, you can run   
 ```yarn test ```  
 
-If you would like to run the integration tests for the project please ensure you have selenium server installed and running  
+
+
+If you would like to run the integration tests locally for the project please be sure to first have a running copy of the frontend and ManageIQ backend running locally.  If you don't have them currently running  
```

Comment:
To run the SUI integration tests,  first confirm that the UI and mock api are both running (check to make sure ManageIQ API is **not** running as this will interfere with the mock API)